cross/Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG CROSS_BASE_IMAGE
+FROM $CROSS_BASE_IMAGE
+
+RUN rm -rf /usr/local/go;
+RUN curl -L https://go.dev/dl/go1.20.5.linux-amd64.tar.gz | tar -xz -C /usr/local
```

Comment:
nice, that's way cleaner and safer without the side effects :)

Reply:
Agreed. Thank you for nudging me to look beyond the first solution I found 👍🏻